feature/policy_forwarding/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,209 @@
+
+		- IPv4Prefix2/24 to IPv4Prefix7/24 at a rate of 100 packets/sec
+		- IPv6Prefix2/64 to IPv6Prefix7/64 at a rate of 100 packets/sec
+
+		- IPv4Prefix3/24 to IPv4Prefix8/24 at a rate of 100 packets/sec
+		- IPv6Prefix3/64 to IPv6Prefix8/64 at a rate of 100 packets/sec
+
+		- IPv4Prefix4/24 to IPv4Prefix9/24 at a rate of 100 packets/sec
+		- IPv6Prefix4/64 to IPv6Prefix9/64 at a rate of 100 packets/sec
+
+		- IPv4Prefix5/24 to IPv4Prefix10/24 at a rate of 100 packets/sec
+		- IPv6Prefix5/64 to IPv6Prefix10/64 at a rate of 100 packets/sec
+
+	Flows from ATE:Port2 to ATE:Port1
+		- IPv4Prefix6/24 to IPv4Prefix1/24 at a rate of 100 packets/sec
+		- IPv6Prefix6/64 to IPv6Prefix1/64 at a rate of 100 packets/sec
+
+		- IPv4Prefix7/24 to IPv4Prefix2/24 at a rate of 100 packets/sec
+		- IPv6Prefix7/64 to IPv6Prefix2/64 at a rate of 100 packets/sec
+
+		- IPv4Prefix8/24 to IPv4Prefix3/24 at a rate of 100 packets/sec
+		- IPv6Prefix8/64 to IPv6Prefix3/64 at a rate of 100 packets/sec
+
+		- IPv4Prefix9/24 to IPv4Prefix4/24 at a rate of 100 packets/sec
+		- IPv6Prefix9/64 to IPv6Prefix4/64 at a rate of 100 packets/sec
+
+		- IPv4Prefix10/24 to IPv4Prefix5/24 at a rate of 100 packets/sec
+		- IPv6Prefix10/64 to IPv6Prefix5/64 at a rate of 100 packets/sec
+
+
+
+## Procedure
+### PF-1.6.1: [Baseline] Default VRF for all flows with regular traffic profile
+
+In this case ATE:Port2 simulates the regular flows as stated above.
+  * ATE:Port2 sends following IPv4 and IPv6 flows:
+    * IPv4Prefix6/24 to IPv4Prefix1/24 
+    * IPv4Prefix7/24 to IPv4Prefix2/24
+    * IPv4Prefix8/24 to IPv4Prefix3/24 
+    * IPv4Prefix9/24 to IPv4Prefix4/24 
+    * IPv4Prefix10/24 to IPv4Prefix5/24
+    * IPv6Prefix6/64 to IPv6Prefix1/64 
+    * IPv6Prefix7/64 to IPv6Prefix2/64
+    * IPv6Prefix8/64 to IPv6Prefix3/64 
+    * IPv6Prefix9/64 to IPv6Prefix4/64 
+    * IPv6Prefix10/64 to IPv6Prefix5/64    
+
+    * Expectations:
+        * All traffic must be successful and there should be 0 packet loss. <br><br><br>
+
+### PF-1.6.2: Traffic from ATE:Port2 to ATE:Port1 Prefix 1 migrated to Non-Default VRF using the VRF selection policy
+  * ATE:Port2 sends following IPv4 and IPv6 flows:
+    * IPv4Prefix6/24 to IPv4Prefix1/24 
+    * IPv4Prefix7/24 to IPv4Prefix2/24
+    * IPv4Prefix8/24 to IPv4Prefix3/24 
+    * IPv4Prefix9/24 to IPv4Prefix4/24 
+    * IPv4Prefix10/24 to IPv4Prefix5/24
+    * IPv6Prefix6/64 to IPv6Prefix1/64 
+    * IPv6Prefix7/64 to IPv6Prefix2/64
+    * IPv6Prefix8/64 to IPv6Prefix3/64 
+    * IPv6Prefix9/64 to IPv6Prefix4/64 
+    * IPv6Prefix10/64 to IPv6Prefix5/64
+  * 
+  * Expectations:
+    * To validate the prefixes advertised by ATE:Port1 are received on ATE:Port2 and vice versa. 
+    * Traffic for Prefix 1 received from ATE:Port2 once punted to non-defailt VRF by the VRF selection policy, must be received by ATE:Port1
+    * Traffic sent by ATE:Port2 must be routed to ATE:Port1 via the DEFAULT VRF in the DUT.
```

Comment:
Last question for you.  How should one verify the traffic is sent using a particular network-instance (vrf?)?   If querying the DUT for this verification, please specify the OC paths needed and what the validation operation is.

Reply:
Any counters being used to verify should be added to the "OpenConfig Path and RPC Coverage" section in yaml below.

Reply:
I added the counters to verify and also some text which mentions to verify.